plugin/gen.go

```diff
@@ -20,6 +20,5 @@
 
 package plugin
 
-//go:generate thriftrw --generate-plugin-api api.thrift
+//go:generate thriftrw --pkg-prefix=go.uber.org/thriftrw/plugin --generate-plugin-api api.thrift
```

Comment:
this looks like it will ripple. will all projects need to add this flag to their thrift gen lines?

Reply:
No, only projects using Go modules need to specify pkg-prefix. All
internal projects already provide it. This change affects ThriftRW's
invocations of itself because we can no longer assume it's on GOPATH.
In the future, when we use go/packages in ThriftRW, this will be
unnecessary again.